src/view/commands/submit_request.rs

```diff
@@ -109,6 +116,11 @@ where
             StyledStr::from("STATUS: ").with_text_style(TextStyle::Bold),
             StyledStr::from(&response_status),
             StyledStr::from(&response_status_message_styled),
+            StyledStr::from("    "),
+            StyledStr::from(&response_time)
+                .with_text_style(TextStyle::Bold)
+                .with_color_bg(Color::White)
+                .with_color_text(Color::Magenta),
         ];
```

Comment:
Even though I thought about adding some colors to these elements, upon further review now, it would be better to use normal text, without anything at all. With these colors, it doesn't look visually appealing. This white color gives too much attention to this information (which is not the most important information in output), and this magenta doesn't match the rest of the colors.

Reply:
<img width="307" alt="Screenshot 2024-02-19 at 9 17 37 PM" src="https://github.com/talis-fb/TReq/assets/42691057/5f1083fd-f1c7-4cf0-a320-d7de7c4d81a6">
Like this

Reply:
Perfect!

Reply:
@SummerGram 
It also would be better format with a `.` for thousands values. Like `1.563 MS` instead `1563 MS` for nice view.